### ISSUE TITLE
[SF_ACC][1.2.0][DynamicPageTitle] Items of overflow button cannot be navigated using arrow keys

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents-react#3704]
**Describe the bug**
There are two accessibility issues of items in overflow button:
1. Items in popover of overflow button cannot be navigated using arrow keys;
2. The item count(e.g. 1 of 2) is not announced by screen reader.

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--with-custom-overflow-button

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--with-custom-overflow-button
3. Click on Overflow button
4. Cannot navigate items in popover using arrow keys
5. The item count is not announced by screen reader

**Expected behavior**
1. Items in popover of overflow button can be navigate using arrow keys;
2. The item count(e.g. 1 of 2) is announced by screen reader.

**Screenshots**

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.8.0
`@ui5/webcomponents-react` version: 1.2.0
Operating System: macOS Monterey
Browser: Chrome

**Additional context**

**Comment by Lukas742** [comment]
Hi @i323808 

the toolbars are implemented according to the SAPUI5 implementation and UX guidelines. 

- [SAPUI5 DynamicPage](https://sapui5.hana.ondemand.com/#/entity/sap.f.DynamicPage/sample/sap.f.sample.DynamicPageFreeStyle)
- [OverflowToolbar](https://sapui5.hana.ondemand.com/#/entity/sap.m.OverflowToolbar/sample/sap.m.sample.OverflowToolbarSimple)

We also had an internal discussion about this topic and the reason behind not offering a keyboard navigation is, that the toolbar also supports input components, for which the navigation would break. That's why only the default tab navigation is used here. 

By leveraging the respective toolbar props (`actionsToolbarProps` or `navigationActionsToolbarProps`), you still have the option to configure the toolbars inside the `DynamicPageTitle` yourself to achieve the desired behavior, but as this would create a discrepancy between the React implementation and the SAPUI5 implementation, I wouldn't recommend it.

I'm going to close this issue now as the SAPUI5 implementation would need to be adapted first and we have to wait for what our accessibility experts have to say about that. 
Please feel free to add a comment if you have new information about this.

**Comment by Lukas742** [comment]
Hi @OliKei 

I'm going to check with our a11y experts and let you know of the outcome. Depending on it we will then reopen this issue.

**Comment by Lukas742** [comment]
Hi @OliKei 

I just heard back from our UX/A11y experts and there are still open topics for discussion (e.g. how to handle Select, ComboBox, etc.) so until they are clarified I'll keep the issue closed, but reopen it once they are plans for arrow navigation.

However you could still implement that behavior yourself (as mentioned above).
